core/src/main/java/org/apache/spark/io/ReadAheadInputStream.java

```diff
@@ -42,7 +45,7 @@
  */
 public class ReadAheadInputStream extends InputStream {
 
-  private static final Logger logger = LoggerFactory.getLogger(ReadAheadInputStream.class);
+  private static final Logger LOGGER = LoggerFactory.getLogger(ReadAheadInputStream.class);
```

Comment:
Let's capitalize variables of type `static final`.

Reply:
I would suggest avoiding such unnecessary change.
In https://www.slf4j.org/api/org/slf4j/Logger.html, the example code is using `logger` as well.

Reply:
Okay